openmetadata-ui/src/main/resources/ui/src/components/TriggerReIndexing/TriggerReIndexing.component.tsx

```diff
@@ -0,0 +1,201 @@
+/*
+ *  Copyright 2023 Collate.
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+import { AxiosError } from 'axios';
+import { useWebSocketConnector } from 'components/web-scoket/web-scoket.provider';
+import {
+  ELASTIC_SEARCH_INDEX_ENTITIES,
+  ELASTIC_SEARCH_INITIAL_VALUES,
+} from 'constants/elasticsearch.constant';
+import { ELASTIC_SEARCH_RE_INDEX_PAGE_TABS } from 'enums/ElasticSearch.enum';
+import { isEqual } from 'lodash';
+import React, { useCallback, useEffect, useMemo, useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import { useParams } from 'react-router-dom';
+import {
+  getBatchJobReIndexStatus,
+  getStreamJobReIndexStatus,
+  reIndexByPublisher,
+  stopBatchJobReIndex,
+} from 'rest/elasticSearchReIndexAPI';
+import { SOCKET_EVENTS } from '../../constants/constants';
+import { CreateEventPublisherJob } from '../../generated/api/createEventPublisherJob';
+import {
+  EventPublisherJob,
+  RunMode,
+} from '../../generated/system/eventPublisherJob';
+import { getJobDetailsCard } from '../../utils/EventPublisherUtils';
+import { showErrorToast, showSuccessToast } from '../../utils/ToastUtils';
+import ReIndexAllModal from './ElasticSearchReIndexModal.component';
+
+function TriggerReIndexing() {
+  const { t } = useTranslation();
+  const { fqn } = useParams<{ fqn: string }>();
```

Comment:
```suggestion
  const { tab } = useParams<{ tab: string }>();
```

Reply:
We are already using the tab parameter in the URL to indicate the menu item selected in the settings left panel, and also the path with the fqn parameter is already in use in Teams page